components/description-list/README.md

```diff
@@ -10,16 +10,20 @@ A Description List displays information in a vertical list of key-value pairs. C
 ```
 
 ## Best Practices
+Use a Description List to display static information that pairs property names and values. In the standard HTML description list `<dl>` element these are called terms `<dt>` and definitions `<dd>`.
```

Comment:
cc: @geurts 

This is the first time we've had text preceding the best practices.

This surfaced a minor spacing issue which I've tried addressing in https://github.com/BrightspaceUI/documentation/pull/1048.

![image](https://user-images.githubusercontent.com/13937038/188715315-11166e67-f1c9-4932-b556-7c2ece426a27.png)

Reply:
I wonder if the text in _Best Practices_ would fit better under another section. I'll let @geurts chime in on that.